src/main/java/com/blogspot/mydailyjava/weaklockfree/WeakConcurrentMap.java

```diff
@@ -91,7 +95,7 @@ public WeakConcurrentMap(boolean cleanerThread, boolean reuseKeys) {
     public V get(K key) {
         if (key == null) throw new NullPointerException();
         V value;
-        final LatentKey<K> latentKey = getKey(key);
```

Comment:
You don’t seem to like final local variables ;)
Is this because of readability or for other reasons?

Reply:
It has no effect and I find it distracting. I avoid it for that reason.